src/chainparams.cpp

```diff
@@ -103,6 +103,7 @@ class CMainParams : public CChainParams {
         nLastPOWBlock = 0x7fffffff;
         nPreStabilityRewardEnsuranceBlock = 935000;
         nStabilityForkBlock = 940000;
+        nThreeOhFix = 975000;
```

Comment:
this block needs to be changed before we release...

need to look at WHERE we are and decide then on what release to do.